test/FunctionalTests/Microsoft.OData.Core.Tests/ODataMultipartMixedBatchReaderTests.cs

```diff
@@ -205,6 +206,42 @@ public void ReadMultipartMixedBatchRequest()
             Assert.Empty(verifyResourceStack);
         }
 
+        [Fact]
+        public void ReadMultipartMixedBatchRequestWthoutContentId()
+        {
+            var payload = Regex.Replace(batchRequestPayload, "Content-ID: .", "");
+            Assert.NotEqual(payload, batchRequestPayload);

```

Comment:
Maybe it would be more valuable to assert the `payload` does not contained a `Content-ID`.